### PR TITLE
NETOBSERV-132 improved table display

### PR DESCRIPTION
**Pull request by jpinsonneau**
- table display sizes (compact / normal / large)
image

  - Compact :
image
  - Normal :
image
  - Large :
image

- tooltips on truncated texts / namespaces / pods
image

- default columns sizes
image

Also unselected adresses & protocol by default as we discussed previously

### CODE REVIEW

File: web/locales/en/plugin__network-observability-plugin.json [hunk 1]
```diff
@@ -8,6 +8,9 @@
   "Cancel": "Cancel",
   "At least one column must be selected": "At least one column must be selected",
   "Save": "Save",
+  "Compact": "Compact",
```
Comment (jotak) [inline web/locales/en/plugin__network-observability-plugin.json:11]: oh, you already implemented this, cool! :-)

File: web/src/components/netflow-table.tsx [hunk 2]
```diff
@@ -22,10 +22,11 @@ import { Column } from '../utils/columns';
 const NetflowTable: React.FC<{
   flows: ParsedStream[];
   columns: Column[];
+  size: string;
```
Comment (jotak) [inline web/src/components/netflow-table.tsx:25]: there could also be a union type here (`'s' | 'm' | 'l'`) to enforce stronger typing, like you did for `width` below?
this pattern is widely used in patternfly too (e.g. https://github.com/patternfly/patternfly-react/blob/main/packages/react-core/src/components/Text/Text.tsx#L22) I guess it's a good practice to adopt too

File: web/locales/en/plugin__network-observability-plugin.json (hunk 1 above)
Comment (jpinsonneau) [reply web/locales/en/plugin__network-observability-plugin.json:11]: yes it was a quick win :1st_place_medal:

**Comment by jpinsonneau** [comment]
> there's a bunch of conflicts with my merged PR

yes I've just done that + added Sizes types

**Review by jotak — CHANGES_REQUESTED** [review]
Looks like there's a bug, in the column managing popup, clicking on "restore default columns" doesn't actually set the defaults, but selects everything instead

File: web/src/components/display-dropdown.tsx [hunk 3]
```diff
@@ -0,0 +1,44 @@
+import * as React from 'react';
+import { useTranslation } from 'react-i18next';
+import { Dropdown, DropdownToggle, DropdownItem } from '@patternfly/react-core';
+import { ThIcon } from '@patternfly/react-icons';
+import * as _ from 'lodash';
+
+export type Sizes = 's' | 'm' | 'l';
```
Comment (jotak) [inline web/src/components/display-dropdown.tsx:7]: thanks! nit: can you just rename to `Size` ? (as a variable of that type has only one size)

**Comment by jpinsonneau** [comment]
> Looks like there's a bug, in the column managing popup, clicking on "restore default columns" doesn't actually set the defaults, but selects everything instead

Good catch, I have also updated [NETOBSERV-125](https://issues.redhat.com/browse/NETOBSERV-125) accordingly

**Review by jotak — APPROVED** [review]
lgtm, thanks!

**Pull request merged**